main.go

```diff
@@ -29,17 +30,46 @@ func NewJoke(id, title, body string, score int) Joke {
 	return Joke{id, title, body, score}
 }
 
+// Page struct
+
+type Page struct {
```

Comment:
I believe a client should know the total amount of records, to know how many pages could be shown/requested